src/main/java/com/sun/tdk/signaturetest/SignatureTest.java

```diff
@@ -374,7 +383,8 @@ private boolean parseParameters(String[] args) {
         parser.addOption(UPDATE_FILE_OPTION, OptionInfo.option(1), optionsDecoder);
         parser.addOption(MODE_OPTION, OptionInfo.option(1), optionsDecoder);
         parser.addOption(ALLPUBLIC_OPTION, OptionInfo.optionalFlag(), optionsDecoder);
-
+        parser.addOption(EXCLUDE_JDK_CLASS_OPTION, OptionInfo.optionVariableParams(1, OptionInfo.UNLIMITED), optionsDecoder);
```

Comment:
This option only influences only command line interface. Neither integration with Maven or Ant! Are you running your checks from command line only?

Reply:
> This option influences only command line interface. Neither integration with Maven or Ant! Are you running your checks from command line only?

The Jakarta EE Platform TCK is tightly integrated with sigtest classes for running the signature tests via code like https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/src/com/sun/ts/tests/signaturetest/SigTestDriver.java#L136

I'll give feedback on what I am seeing now with #3 on the pr.